src/dbxdeploy/_config/config.yaml

```diff
@@ -13,6 +13,7 @@ parameters:
         patterns: ['**/*.py']
         consumers_patterns: ['**/*.consumer.py']
         master_package_notebook_path: '%dbxdeploy.source.notebooks.base_dir%/app/install_master_package.py'
+        cell_separator: 'percent' # command or percent
```

Comment:
Just note: I would choose "command" as default value for now as this is still a bit experimental feature and we want to make it backward compatible.